langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -357,6 +512,18 @@ public isolated function forEach(xml<ItemType> x, @isolatedParam function(ItemTy
 #
 # Each item is represented as a singleton value.
 #
+# ```ballerina
+# xml codingBooks = xml `<script>JS</script><code>Ballerina</code><script>Perl</script><code>Python</code>`;
+# 
+# codingBooks.filter(function(xml xmlItem) returns boolean {
+#     if xmlItem is xml:Element {
+#         return (<xml:Element>xmlItem).getName() == "code";
+#     }
+# 
+#     return false;
+# }) ⇒ <code>Ballerina</code><code>Python</code>
```

Comment:
```suggestion
# xml books = xml `<script>JS</script><code>Ballerina</code><script>Perl</script><code>Python</code>`;
# books.filter(x => x is xml:Element && x.getName() == "code") ⇒ <code>Ballerina</code><code>Python</code>
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/8bceaabd7a1a2202d2929fee34d1e15e2f2b9b18